syntax = "proto3";
package stream;





/*
ReplaceAggregateStates返回
*/
message ReplaceAggregateStatesResponse {
message States {
    /*
    状态数据id
    */
    string id = 1 ;
    /*
    状态数据版本
    */
    int32 version = 2 ;
}
        
    
    /*
    推送的状态数据列表
    */
    repeated States states = 1 ;
}

/*
ReplaceAggregateStatesApi返回
*/
message ReplaceAggregateStatesResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    stream.ReplaceAggregateStatesResponse data = 4 ;
}

